plugins/LadspaEffect/caps/interface.cc

```diff
@@ -29,7 +29,7 @@
 	(2541 - 2580 donated to artemio@kdemail.net)
 */
 
-#include <sys/time.h>
+// #include <sys/time.h> this header is unnecessary and breaks msvc
```

Comment:
The note should be commented, but I don't believe it to be accurate.  You indicated that it was likely leftover from `seed()`.  The comment should mention that so that future contributors knows why it's still here, but commented out.

Reply:
Yeah i guess i need to be more specific in my comments. Will fix it next commit. Don't wanna put every minor line change on seperate commits like i did just now.

Reply:
I pushed the change now.